LevelUp.Pos.ProposedOrders.Tests/UpdateSpendAmountCompleteOrderTests.cs

```diff
@@ -1,150 +0,0 @@
-﻿#region Copyright (Apache 2.0)
```

Comment:
I removed the `CalculateAdjustedSpendAmountCompleteOrder` in favor of reusing the `CalculateAdjustedSpendAmount` method and passing in `totalOutstandingAmountWithDiscount` in place of `totalOutstandingAmount` for the "Complete Order Calculator".